SDWebImage/Core/SDWebImageCacheKeyFilter.h

```diff
@@ -26,6 +26,9 @@ typedef NSString * _Nullable(^SDWebImageCacheKeyFilterBlock)(NSURL * _Nonnull ur
  */
 @interface SDWebImageCacheKeyFilter : NSObject <SDWebImageCacheKeyFilter>
 
+- (nonnull instancetype)init NS_UNAVAILABLE;
```

Comment:
You can the simiar thing (this called code garden, no functional changes) for RequestModifer, ResponseModifier as well

They introduced based on the same design to use a class to wrap a objc block (for Swift user only, because they're suck at passing Swift closure into NSMutableDictionary, we called SDWebImageContext)